src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/FareProducts/PurchaseTab.tsx

```diff
@@ -67,7 +74,24 @@ export const PurchaseTab: React.FC<Props> = ({navigation}) => {
             }
           />
         )}
+
+        {showTipsAndInformation && (
+          <TipsAndInformationTile
+            onPress={() => {
+              navigation.navigate('Root_TipsAndInformation');
+            }}
+            testID="tipsAndInformation"
+          />
+        )}
         <AvailableFareProducts onProductSelect={onProductSelect} />
+        {showTicketAssistant && (
+          <TicketAssistantTile
```

Comment:
A padding must be added at the bottom and perhaps less at the top.

<img width="353" alt="image" src="https://user-images.githubusercontent.com/43450882/227893001-74365418-6d7b-40e1-8918-ba8cdd979131.png">